### DISCUSSION TITLE
How do you specify the checkpoint with --ckpt

### DISCUSSION THREAD

**Discussion by econundrum** [vladmandic/sdnext#2133] [General]
Hi I'm using windows I'm trying to setup different shortcuts for sdxl and sd 1.5 and want to specify different ckpts in the command line. I've tried various variations on the path and name of the ckpt I want to use both relative and absolute as well as just the name and it can never find it.

Has anyone got an example of using --ckpt so I can see what I'm doing wrong.

**Comment by vladmandic** [comment]
not sure what's the problem, i just tried and works with both absolute and relative path to a model.

**Reply by econundrum** [reply]
Could you show me the command line your using so I can see what I'm doing wrong.

**Comment by brknsoul** [comment]
Right-click `webui.bat` and click "Create shortcut".
Right-click this new shortcut and click Properties.
In the Target field add to the end

`--ckpt models\Stable-Diffusion\ExactNameOfModel.safetensors`
along with any regular arguments.

For example, I'm running on an AMD GPU, and I want to run Dreamshaper 8, so mine looks like this;

`E:\automatic\webui.bat --use-directml --medvram --ckpt models\Stable-diffusion\dreamshaper_8.safetensors`

**Reply by econundrum** [reply]
Thanks I see the error I made had a \ in front of models, rookie error

**Reply by econundrum** [reply]
I actually tried giving a full path in the short cut once once e.g. D:\SD3\automatic\webui.bat --medvram --backend original --upgrade --ckpt D:\SD3\automatic\models\Stable-diffusion\realisticVisionV50_v50VAE.safetensors

That didn't work couldn't find the model I'm assuming the relative one will as you've put it since I did have that extra \ when I tried that.
